RNA-seq/setup/04-nb_cell_line_tximeta.nb.html

```diff
@@ -503,14 +510,16 @@ <h2>Import expression data with <code>tximeta</code></h2>
 <!-- rnb-source-begin eyJkYXRhIjoiYGBgclxudHhpX2RhdGEgPC0gdHhpbWV0YShjb2xkYXRhKVxuYGBgIn0= -->
 <pre class="r"><code>txi_data &lt;- tximeta(coldata)</code></pre>
 <!-- rnb-source-end -->
-<!-- rnb-output-begin eyJkYXRhIjoiaW1wb3J0aW5nIHF1YW50aWZpY2F0aW9uc1xucmVhZGluZyBpbiBmaWxlcyB3aXRoIHJlYWRfdHN2XG4xIDIgMyA0IDUgNiA3IDggOSAxMCAxMSAxMiAxMyAxNCAxNSAxNiAxNyAxOCAxOSAyMCAyMSAyMiAyMyAyNCAyNSAyNiAyNyAyOCAyOSAzMCAzMSAzMiAzMyAzNCAzNSAzNiAzNyAzOCAzOSBcbmZvdW5kIG1hdGNoaW5nIHRyYW5zY3JpcHRvbWU6XG5bIEVuc2VtYmwgLSBIb21vIHNhcGllbnMgLSByZWxlYXNlIDk1IF1cbmxvYWRpbmcgZXhpc3RpbmcgRW5zRGIgY3JlYXRlZDogMjAyMS0wMi0xNyAxNTo0MDo0NlxubG9hZGluZyBleGlzdGluZyB0cmFuc2NyaXB0IHJhbmdlcyBjcmVhdGVkOiAyMDIxLTAyLTE3IDE1OjQwOjQ5XG4ifQ== -->
+<!-- rnb-output-begin eyJkYXRhIjoiaW1wb3J0aW5nIHF1YW50aWZpY2F0aW9uc1xucmVhZGluZyBpbiBmaWxlcyB3aXRoIHJlYWRfdHN2XG4xIDIgMyA0IDUgNiA3IDggOSAxMCAxMSAxMiAxMyAxNCAxNSAxNiAxNyAxOCAxOSAyMCAyMSAyMiAyMyAyNCAyNSAyNiAyNyAyOCAyOSAzMCAzMSAzMiAzMyAzNCAzNSAzNiAzNyAzOCAzOSBcbmZvdW5kIG1hdGNoaW5nIGxpbmtlZCB0cmFuc2NyaXB0b21lOlxuWyBFbnNlbWJsIC0gRGFuaW8gcmVyaW8gLSByZWxlYXNlIDk1IF1cbmxvYWRpbmcgZXhpc3RpbmcgRW5zRGIgY3JlYXRlZDogMjAyMS0wMi0yNCAxNjozNDo0NFxubG9hZGluZyBleGlzdGluZyB0cmFuc2NyaXB0IHJhbmdlcyBjcmVhdGVkOiAyMDIxLTAyLTI0IDE2OjM0OjQ1XG5FcnJvciBpbiBjaGVja0Fzc2F5czJUeHBzKGFzc2F5cywgdHhwcykgOiBcbiAgbm9uZSBvZiB0aGUgdHJhbnNjcmlwdHMgaW4gdGhlIHF1YW50aWZpY2F0aW9uIGZpbGVzIGFyZSBpbiB0aGUgR1RGXG4ifQ== -->
 <pre><code>importing quantifications
 reading in files with read_tsv
 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 
-found matching transcriptome:
-[ Ensembl - Homo sapiens - release 95 ]
-loading existing EnsDb created: 2021-02-17 15:40:46
-loading existing transcript ranges created: 2021-02-17 15:40:49</code></pre>
+found matching linked transcriptome:
+[ Ensembl - Danio rerio - release 95 ]
+loading existing EnsDb created: 2021-02-24 16:34:44
+loading existing transcript ranges created: 2021-02-24 16:34:45
+Error in checkAssays2Txps(assays, txps) : 
+  none of the transcripts in the quantification files are in the GTF</code></pre>
```

Comment:
You will want to revert the changes in this notebook (and probably rerun it to fix your data files), as it seems to have started using the wrong reference during the troubled times. You can use this to see what happens when you have the wrong transcriptome though!

Also interesting to note that linkedTxomes override the provided ones, but that kind of makes sense.